services/email_verification.js

```diff
@@ -0,0 +1,79 @@
+/**
+* @file email_verification.js
+* @author Capuccino
+* @license BSD-3-Clause
+*/
+
+const micro = require("micro");
+const {json} = micro;
+const redis = require("redis");
+const Redite = require("redite");
+const config = require("./email.config");
+const snowflake = require("snowflake-codon");
+const client = redis.createClient({
+    host: config.redis.host,
+    password: config.redis.password
+});
+const db = new Redite({client});
+const idGen = new snowflake(12, Math.floor(Math.random() * 1129), 2018, 200);
```

Comment:
IDK if snowflakes are ok for this, I haven't really paid any attention to much conversations so maybe you guys agreed on this, but perhaps replace this with a hash of some user info to ensure randomness + minimise the chances of guessing it.